cgmanifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "Registrations": [
+        {
+            "Component": {
+                "Type": "other",
+                "Other": {
+                    "Name": "cmake module for adding code coverage",
+                    "Version": "any version",
+                    "DownloadUrl": "https://github.com/bilke/cmake-modules/blob/master/CodeCoverage.cmake"
```

Comment:
How do we plan to keep things in sync (if we have to)? Looks like there were already some recent changes made here.
https://www.diffchecker.com/c7TO8u5I

Reply:
we don't need to.
License allows changes to the original.  We don't need to keep latest version. We can also remove or make changes to it, just keeping the mention in the Notice.